src/app/work-item/work-item-board/work-item-board.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule }           from '@angular/core';
+import { CommonModule }       from '@angular/common';
+import { DragulaModule } from 'ng2-dragula/ng2-dragula';
+
+import { AlmIconModule } from './../../shared-component/icon/almicon.module';
+import { WorkItemBoardComponent }   from './work-item-board.component';
+import { BoardRoutingModule }   from './work-item-board-routing.module';
+//pipes
+
+import { AlmFilterBoardList } from './../../pipes/alm-board-filter.pipe';
+import { CountPipe } from 'ng2bln-count-pipe';
```

Comment:
This is a style thing. We are trying to keep the external libraries at the top with angular first.